apt.html

```diff
@@ -65,6 +65,19 @@ <h2>{% t apt.apt.repo %}</h2>
 
             <pre><code># apt-get install bitcoinxt</code></pre>
 
+	    <p>{% t apt.apt.locatepassword %}</p>
+
+	    <pre><code># cat /etc/bitcoinxt/bitcoin.conf</code></pre>
+
+	    <p>{% t apt.apt.run %}</p>
+
+	    <pre><code># bitcoinxt-cli -rpcuser=bitcoinxt -rpcpassword=AutoGeneratedPasswordFromBitcoinDotConfGoesHere getblockchaininfo</code></pre>
+
+	    <p>{% t apt.apt.prune %}</p>
+
+            <pre><code>prune=5000</code></pre>
+            <pre><code>maxuploadtarget=6000</code></pre>
```

Comment:
This is one specific use case of many, but common and worth mentioning. 

Adding common use cases here will put a lot of work on our translators. How about creating a "Common configurations" article in our wiki, and link to it from here?

Also, we don't support maxuploadtarget, we do however have parameters `receiveavg`, `receiveburst`, `sendburst` and `sendavg`, that unlike `maxuploadtarget` _actually_ allows you to predictably reduce bandwith usage. These would deserve an mention in an article.